test/module/libs/util/util_test.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <cstdlib>
+
+#include "util/network.hpp"
+#include "util/string.hpp"
+
+
+using namespace std::literals::string_literals;
+using namespace iroha;
+
+TEST(String, IsPrintable) {
```

Comment:
Again please use given when then terminology

Reply:
These tests are trivial, I guess there is no need to change them to "given->when->then" terminology

Reply:
there is a need to leave at least a small meaningful comment to each. Also, I've seen other tests above, it seems to me that each test should be separated to individual, and in other file. Do not put everything in util test — otherwise it will become new ametsuchi_test

Reply:
Imo the test name is more than enough for these